src/Extractor/HttpClient.php

```diff
@@ -52,19 +52,7 @@ public function __construct(Client $client, $config = array(), LoggerInterface $
             // to see if returned content type matches $headerOnlyTypes.
             'header_only_clues' => array('pdf', 'mp3', 'zip', 'exe', 'gif', 'gzip', 'gz', 'jpeg', 'jpg', 'mpg', 'mpeg', 'png', 'ppt', 'mov'),
             // User Agent strings - mapping domain names
-            'user_agents' => array(
-                'lifehacker.com' => 'PHP/5.2',
-                'gawker.com' => 'PHP/5.2',
-                'deadspin.com' => 'PHP/5.2',
-                'kotaku.com' => 'PHP/5.2',
-                'jezebel.com' => 'PHP/5.2',
-                'io9.com' => 'PHP/5.2',
-                'jalopnik.com' => 'PHP/5.2',
-                'gizmodo.com' => 'PHP/5.2',
-                '.wikipedia.org' => 'Mozilla/5.2',
-                '.fok.nl' => 'Googlebot/2.1',
-                'getpocket.com' => 'PHP/5.2',
-            ),
```

Comment:
If we remove that, we should be sure that these values are in the config file for each domain.